azure/init.sh

```diff
@@ -40,7 +40,7 @@ su - ${TARGET_USER} <<'EOSU'
  # add pg and local binaries to the path
   echo "export PATH=${HOME}/pg-latest/bin/:${HOME}/.local/bin/:$PATH" >> ${HOME}/.bash_profile
 
-  cd ${HOME} && git clone https://github.com/citusdata/test-automation.git
+  cd ${HOME} && git clone --branch all_test_try https://github.com/citusdata/test-automation.git
```

Comment:
If we want to change configs, timings we will need to clone that branch here. So we should probably pass the branch here from the circleci job if possible. Otherwise it is kind of tedious to update this and it can be easily forgotten.

Reply:
Yes, that'd be great. If not, we should document to change the branch name here as well.

Reply:
I have changed it to be done automatically. If it is run in local it will use `master` by default if not it will use the branch name.